bin/initial.sh

```diff
@@ -1,5 +1,13 @@
-#!/bin/bash
+#!/usr/bin/env bash
+# The following script is run when deploying / upgrading a machine. It should be run from the ansible-playbook directory.
+# Ensure that the hostname that you run this on has already been added to the inventory file.
+
+if [ "$#" -ne 1 ]; then
```

Comment:
Can you run shellcheck on this file, I think this probably should be "$@"

Reply:
Is the intention to use this script on multiple hostnames at the same time?  $@ gives the argument as an array (which is good for multiple input arguments), $# is the number of arguments given to the script, so if it gets a second argument it fails

Reply:
Ah right, sorry, I thought that was "$*" which is the whole command line.